helm/conjur-app-deploy/charts/app-secrets-provider-p2f-injected/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for the Application with Secrets Provider init container in Push-to-File mode Helm chart
+# This is a YAML-formatted file.
+
+global:
```

Comment:
Should there be a JSON schema and/or unit tests associated with this chart? Seems that the majority have them